tests/unittests/GraphOptzTest.cpp

```diff
@@ -488,6 +492,7 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvButVarReused) {
   EXPECT_EQ(varFilter, CV->getFilter());
   EXPECT_TRUE(llvm::isa<BatchNormalizationNode>(optimizedF_ret->getInput()));
 
+
```

Comment:
Why did you add this extra blank line?

Reply:
My bad. I thought it was added by a ./utils/format.sh.
Will fix.